scripts/local-cluster.sh

```diff
@@ -250,6 +250,10 @@ resolve() {
             resolvectl dns "${link}" 127.0.64.43
           fi
         done
+      elif test -x /usr/sbin/unbound-control; then
```

Comment:
```suggestion
      elif command -v unbound-control &>/dev/null; then
```

Reply:
While it is sub-optimal to depend on the tool being in a specific location, it also might not be in `$PATH` when running as a regular user. 🤔

Reply:
Perhaps `command -v || [[ -x /usr/sbin/... ]]` ? "Why not both" 😁

Reply:
So how about ff6b1062d384ace1be0115942f119fda41509c90

Otherwise I am not sure how to deal with the sudoers `secure_path` setting that is used in some places to override the `$PATH` you get when running `sudo`

Reply:
I'm fine with `command -v || [[ -x /usr/sbin/... ]]`.